src/scicookie/{{cookiecutter.project_slug}}/build-system/poetry/pyproject.toml

```diff
@@ -1,19 +1,33 @@
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
-
 {% if cookiecutter.project_layout == "src" -%}
 {% set package_path = "src/" + cookiecutter.package_slug -%}
 {% else -%}
 {% set package_path = cookiecutter.package_slug -%}
 {% endif -%}
 
-[tool.poetry]
+[project]
 name = "{{ cookiecutter.project_slug }}"
 version = "{{ cookiecutter.project_version }}"  # semantic-release
 description = "{{ cookiecutter.project_short_description }}"
 readme = "README.md"
 authors = ["{{ cookiecutter.author_full_name }} <{{ cookiecutter.author_email }}>"]
+license = "{{ cookiecutter.project_license }}"
+dependencies = [
+{%- if cookiecutter.documentation_engine == 'quarto' or cookiecutter.sphinx_theme == 'pydata' or cookiecutter.jupyter_book_theme == 'pydata-sphinx-theme' %}
+  "python >= 3.9,<4"
```

Comment:
In both, if and else, the output is the same .. so it is not necessary to have if/else

But any way .. the python souldn't be listed in rhe dependencies anymore .. it should be moved to another entry called require-python. Please check examples in the issue for more references.

Reply:
Yes, I had placed it there but I don't know why I later deleted it. I'll put it back